charts/seed-monitoring/charts/prometheus/templates/prometheus.yaml

```diff
@@ -206,6 +209,10 @@ spec:
       - name: blackbox-exporter-config-prometheus
         configMap:
           name: blackbox-exporter-config-prometheus
+      - name: prometheus
```

Comment:
This secret already exists here (see line 203-205). Not needed to mount it again.